### PR TITLE
Setup Enterprise

### PR DESCRIPTION
**Pull request by brdlyptrs**
Consolidates translations and sets up CI for deploy.

- [x] Add Pages
- [x] Deploy Script
- [x] CI Deploy
- [x] Fix Tests

### CODE REVIEW

**Comment by brdlyptrs** [comment]
@unick333 can you please fix the test file to include both the `main.json` and `enterprise.json` files. The idea is to make sure the keys in both files match the the keys in the translation files, and if any are missing to fail. See [here](https://github.com/hCaptcha/website-i18n/blob/master/bin/check_expected_keys.py).

**Comment by e271828-** [comment]
@brdlyptrs it seems like something is wrong in this PR. 

- Why are all the non-enterprise translations deleted?
- You haven't merged the ent + non-ent translations together (this is fine) but in this case you cannot remove the non-ent..

**Comment by brdlyptrs** [comment]
@e271828- non enterprise file name was changed to. `main_keys.json`. I figure in the future we will have json files per page vs entire site for faster loading, this is the initial setup for it.

**Comment by e271828-** [comment]
> @e271828- non enterprise file name was changed to `main_keys.json`, nothing was removed. I figure in the future we will have json files per page vs entire site for faster loading, this is the initial setup for it.

Did you in fact sync this with the on-site JS already?

**Comment by brdlyptrs** [comment]
Yep, all the translations have already been deployed to S3 and are pulled into the current live site. I've even update the JS snippet here and on the website to pull from the new S3 location `website-i18n` along with checking which page it is on to get the correct translation.

**Review by e271828- — APPROVED** [review]

**Comment by e271828-** [comment]
@brdlyptrs Please remove .DS_Store prior to merge.

File: locales/fr/enterprise.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  "pilot-head": "Essayez-nous dans votre environnement sans engagement.",
+  "pilot-text-1a": "Nous sommes des ingénieurs, pas des vendeurs : nous voulons que vous soyez satisfait de votre choix.",
+  "pilot-text-1b": "La meilleure façon d'assurer votre satisfaction est de démontrer que nous répondons à vos besoins.",
+  "pilot-text-1c": "Pratiquement toutes les entreprises qui lancent un projet pilote choisissent finalement nos solutions.",
+  "pilot-text-1d": "Utilisez ce formulaire pour joindre l'un de nos spécialistes techniques. Nous répondons rapidement.",
+  "pilot-text-1e": "Vous pouvez également nous envoyer un e-mail directement.",
+  "pilot-email-details": "Les détails sont ici.",
+  "pilot-f1": "Région",
+  "pilot-f2": "votre nom",
+  "pilot-f3": "votre nom",
```
Comment (e271828-) [inline locales/fr/enterprise.json:137]:
```suggestion
  "pilot-f3": "votre entreprise",
```

**Pull request merged**